module/apmgoredisv8/hook_test.go

```diff
@@ -120,14 +120,14 @@ func TestHookTxPipeline(t *testing.T) {
 			})
 
 			require.Len(t, spans, 1)
-			if _, ok := client.(*redis.Client); ok {
-				// *redis.Client wraps queued commands with MULTI/EXEC
-				// in (*redis.Client) processTxPipeline
-				assert.Equal(t, "MULTI, GET, SET, GET, (empty command), EXEC", spans[0].Name)
-			} else {
-				// *redis.ClusterClient and *redis.Ring don't wrap queued commands
-				// with MULTI/EXEC in (*redis.Client) processTxPipeline
+			if _, ok := client.(*redis.Ring); ok {
+				// *redis.Client wraps queued commands without MULTI/EXEC
+				// in (*redis.Ring) processTxPipeline
```

Comment:
```suggestion
				// *redis.Ring does not wrap queued commands with MULTI/EXEC
				// in (*redis.Ring) processTxPipeline
```

I think? IIUC the only change is that ClusterClient now *does* wrap them.

Reply:
@axw thanks for reminding me about the comment, already fixed it.